### DISCUSSION TITLE
Cant upload pics

### DISCUSSION THREAD

**Discussion by Champ-ion28** [photoprism/photoprism#4160] [Q&A]
I have downloaded photo prism app from trunas scale charts. I am able to access the Web u.I but when I import a photo it will not show up.And I get errors in the log.See picture below. All of my settings when I set up the account are default except for the password.
![Screenshot_20240407_201659_Chrome](https://github.com/photoprism/photoprism/assets/121738462/7c11750b-2729-4939-b274-3e85ada9a75a)

**Comment by graciousgrey** [comment]
This looks like PhotoPrism does not have permissions to write within the originals folder.

https://docs.photoprism.app/getting-started/troubleshooting/docker/#file-permissions
